.github/workflows/ci-build-tests.yml

```diff
@@ -70,10 +66,6 @@ jobs:
           python-version: '3.x'
       - name: Install python dependencies
         run: python -m pip install --upgrade pip setuptools wheel
-      - name: installing codespell
```

Comment:
todo: Add this back; golangci-lint misspell isn't as affective as I thought it might be